terraform/layer2-k8s/templates/elk-values.yaml

```diff
@@ -145,18 +145,14 @@ filebeat:
             hints.default_config:
               type: container
               paths:
-                - /var/log/containers/*-$${data.kubernetes.container.id}.log
+                - /var/log/containers/*$${data.kubernetes.container.id}.log
 
       processors:
         - drop_event:
             when:
               equals:
                 kubernetes.container.name: "filebeat"
 
-        - add_cloud_metadata:
-        - add_kubernetes_metadata:
```

Comment:
Do we have kubernetes labels without this setting?

Reply:
Yes, autodiscover automatically added kubernetes labels.